### ISSUE TITLE
SurveyActivity: new user(s) layout proposal

### ISSUE THREAD

**Issue by valllllll2000** [akvo/akvo-flow-mobile#670]
Here are some screenshots of how this could look:
Initial look when user is selected:
![screenshot_1490296091](https://cloud.githubusercontent.com/assets/923280/24265677/a2282ec8-1004-11e7-9dc9-8e4a5042e8d4.png)

after pressing on the spinner, the dropdown is displayed:
![screenshot_1490296096](https://cloud.githubusercontent.com/assets/923280/24265718/bd843d06-1004-11e7-930d-025ffe10ddd1.png)

if you select "+user": (same layout as before)
![screenshot_1490296100](https://cloud.githubusercontent.com/assets/923280/24265736/cb0d9a6c-1004-11e7-8eb8-8511afdb07e2.png)

**Comment by janagombitova** [comment]
@valllllll2000 I have a question looking at how to user part works on your device and comparing it to mine. On my devices when I expand the user part, the entire menu expands and I do not get the little square. I personally prefer that the user part expands within the menu, but this is my opinion. Why is my experience different from yours (different devices and Android versions) and how can we make sure it is unified? 

It looks like this for the newer phone: 
![screen shot 2017-03-22 at 11 21 43](https://cloud.githubusercontent.com/assets/12456965/24193371/54b2f442-0ef2-11e7-8eb9-fda64d094850.png)

For the old dinosaur: 
![screen shot 2017-03-22 at 11 23 13](https://cloud.githubusercontent.com/assets/12456965/24193372/56a77bd8-0ef2-11e7-9491-6abebc88aaff.png)

**Comment by valllllll2000** [comment]
@janagombitova so about the diferences between the new and old devices: there is nothing I can do, the system has diferent dialogs and styling for older devices. 

For the way I am displaying the user, this is a proposal to modify the way the user is displayed, to have to similar to some google apps, for example here gmail:

![screenshot_20170324-180104](https://cloud.githubusercontent.com/assets/923280/24305127/30741806-10bc-11e7-868a-80338c4ee08b.png)
although the behaviour that I display is not exactly the same as from the gmail app so I will try to reproduce it and paste the new screenshots here

**Comment by valllllll2000** [comment]
Here are the new options for displaying the user:
![screenshot_1490377432](https://cloud.githubusercontent.com/assets/923280/24306716/d9ce7b9e-10c1-11e7-837e-f03bde96ff30.png)

After clicking on the user it expands:
![screenshot_1490377449](https://cloud.githubusercontent.com/assets/923280/24306731/e219ec7a-10c1-11e7-841a-b342b76b6396.png)

**Comment by valllllll2000** [comment]
and more options for the user display:
![screenshot_1499698126](https://user-images.githubusercontent.com/923280/28024002-ce9cbd26-658f-11e7-9009-c4ba8311fcbb.png)
![screenshot_1499696516](https://user-images.githubusercontent.com/923280/28024001-ce9afa54-658f-11e7-8526-858999760c64.png)

**Comment by janagombitova** [comment]
I personally like the 1st option as we do not have the chance to add an image for a user (currently) so the big extra orange space right now is a waste. Upon expanding you will see the other possible users and the chance to add a new one, right (as shown in older comments)?

**Comment by valllllll2000** [comment]
@janagombitova yes, that is the idea, to have it look similar to gmail app or other google apps.

**Comment by valllllll2000** [comment]
@janagombitova while working on this issue, a question arose:
1) Should we be allowed to delete the current user? I would suggest that no, because that would leave us in an odd state and user having to select another user before being able to continue. I would have the users first create another user and select it and then they can delete the previous.
2) What if we delete the last user, should we be taken to the login screen? This is kind of related to this https://github.com/akvo/akvo-flow-mobile/issues/790

**Comment by janagombitova** [comment]
Hi Valeria, good questions. As this issue is about layout, I have created another one to discuss and improve the deletion of a user and what rules we want to put in place. The issue is here: https://github.com/akvo/akvo-flow-mobile/issues/814

**Comment by valllllll2000** [comment]
Here are some screenshots
![screenshot_2017-08-04_1500](https://user-images.githubusercontent.com/923280/28969700-f488d944-7925-11e7-84ac-cbd8f0777e73.png)
![screenshot_2017-08-04_1459_1](https://user-images.githubusercontent.com/923280/28969701-f4895f40-7925-11e7-8c8f-05f8f92648d0.png)
![screenshot_2017-08-04_1459](https://user-images.githubusercontent.com/923280/28969699-f488495c-7925-11e7-99ac-38b997ff29e1.png)
![screenshot_2017-08-04_1451_1](https://user-images.githubusercontent.com/923280/28969702-f48bf3a4-7925-11e7-99e0-68db6f1f2583.png)

**Comment by janagombitova** [comment]
I assume regression testing this one and testing all the smaller related issues confirm that this change is ready for release. Ready for release
